### DISCUSSION TITLE
想知道怎么查到在哪里用过v2ray

### DISCUSSION THREAD

**Discussion by muziyin** [v2fly/v2ray-core#2674] [Q&A]
我手机丢了，手机里装有v2ray，检查发现有些我没用过的服务器有流量使用。想请教下，怎么通过自己电脑客户端找到用同一个客户端的手机，曾在哪里用过v2rayN。

**Comment by fishBone000** [comment]
如果服务器是自己的，应该可以到服务器上抓流量吧，除此之外应该别无他法了

**Reply by ZXiangQAQ** [reply]
直接生成一个新的UUID，弃用之前的不就行了，整白名单太麻烦了，假如真的有人在使用这个链接，不改链接是封不完的
